Extensions/Firebase/tests/FirebaseCallbackRemover.js

```diff
@@ -0,0 +1,4 @@
+// Remove the firebase first scene loaded callback to not be called by other tests.
+// It wouldn't work as it would be called multiple times,
+// and sometimes with RuntimeGame mocks without extension properties.
+gdjs.callbacksFirstRuntimeSceneLoaded.length = 0;
```

Comment:
Is there a risk that this breaks silently other tests in the future? Could be hard to debug.
Could we make firebase resilient against mutiple "first runtime scene loaded" invocations?

Reply:
Well we could ignore subsequent calls to prevent firebase from crashing, but that would mean the first invocation would set the config and the tests aren't the first callers. I doubt any test could rely on those callbacks anyways as they are often called with incomplete mocks.

Reply:
We could check if the config passed is exactly the same as before, and if yes ignore it. If not, throw an error telling that is unexpected and unsupported :)